hadoop-hdfs-project/hadoop-hdfs-rbf/src/main/java/org/apache/hadoop/hdfs/server/federation/router/async/RouterAsyncClientProtocol.java

```diff
@@ -1086,4 +1093,35 @@ public boolean isMultiDestDirectory(String src) {
 
     return asyncReturn(boolean.class);
   }
+
+  @Override
+  public Path getEnclosingRoot(String src) throws IOException {
+    final Path[] mountPath = new Path[1];
```

Comment:
Here give array which size is 1, what about `Path mountPath = null;`, any concerns here?

Reply:
> Here give array which size is 1, what about `Path mountPath = null;`, any concerns here?

@Hexiaoqiao Sir, thanks for reviewing. if we use `Path mount = null` here, we will encounter syntax error because we will use mount in lambda experssion.